katalogus/views/katalogus.py

```diff
@@ -37,22 +46,29 @@ def get_queryset(self):
         if "filter_options" in self.request.GET:
             filter_options = self.request.GET.get("filter_options")
             queryset = self.filter_queryset(queryset, filter_options)
+        if "sorting_options" in self.request.GET:
+            sorting_options = self.request.GET.get("sorting_options")
+            queryset = self.sort_queryset(queryset, sorting_options)
         return queryset
 
     def filter_queryset(self, queryset, filter_options):
-        if filter_options == "a-z":
-            return queryset
-        if filter_options == "z-a":
-            return list(reversed(queryset))
         enabled = [plugin for plugin in queryset if plugin["enabled"]]
+        disabled = [plugin for plugin in queryset if not plugin["enabled"]]
         if filter_options == "enabled":
             return enabled
-        disabled = [plugin for plugin in queryset if not plugin["enabled"]]
         if filter_options == "disabled":
             return disabled
-        if filter_options == "enabled-disabled":
+
+    def sort_queryset(self, queryset, sort_options):
+        if sort_options == "a-z":
+            return queryset
+        if sort_options == "z-a":
+            return list(reversed(queryset))
+        enabled = [plugin for plugin in queryset if plugin["enabled"]]
```

Comment:
A more pythonic way to do this is to use `sorted`:
e.g.:
```python3
queryset = sorted(queryset, lambda item: not item["enabled"])
```
or 
```python3
queryset = sorted(queryset, lambda item: item["enabled"])
```
for the other way around

Although I'm not sure what the type of query set is because type hint is missing, but I'm guessing it's a `List[Dict[str, Any]]`.

Reply:
I don't really understand what you mean here. `[plugin for plugin in queryset if plugin["enabled"]]` filters the iterable, it doesn't sort it...

Reply:
Done :)